src/testUtils/enzymeHelpers.jsx

```diff
@@ -10,6 +12,22 @@ const getThemeProviderWrappingComponent = (customTheme) => ({ children }) => (
     </ThemeProvider>
 );
 
+export const createMatchMedia = (width) => {
+    return (query) => ({
+        matches: mediaQuery.match(query, { width }),
+        addListener: () => {},
+        removeListener: () => {},
+    });
+};
+
+export const waitForComponentToPaint = async (wrapper) => {
+    await act(async () => {
+        await new Promise((resolve) => setTimeout(resolve, 0));
+
+        wrapper.update();
+    });
+};
```

Comment:
Very nice.  We might consider adding these in HC (`church-management`) as well so we can use them in tests there too.

Reply:
I agree! :D